gradle/publish.gradle

```diff
@@ -100,6 +100,7 @@ configure(javaProjects) { project ->
                 artifact sourcesJar
                 artifact javadocJar
 
+                suppressAllPomMetadataWarnings()
```

Comment:
A side effect of using gradle features.  We dont need to publish the metadata in the pom at this time - so I've disabled the warnings.

Reply:
Done